processor/awscontainerinsightprocessor/README.md

```diff
@@ -0,0 +1,445 @@
+  * find corresponding pod objects from the cache to decorate pod and container metrics
+  * information used to decorate pod and container metrics:
+    1. Pod Owner Reference
+    2. Container Resource Limit
+    3. Container Resource Request
+    4. Container ID and Status
+    5. Pod Status
+    6. Pod Labels
+* service store: 
+  * add the corresponding attribute `Service` to the metrics for the relevant pods
+  * call "List & Watch" for "Endpoint" objects from Kubernetes API server. In this mode, when there is any change to “Endpoint”, agent will get the notification of the change. The benefit is that the agent could get Endpoint information update in time. If the "Endpoint" resources just occasionally slightly changes, the impact to API server is supposed to be minimal.
+  * If customer is not interested in aggregating metrics per ServiceName, we allow them to disable the service decoration through the configuration.
+
+
+## Available Metrics and Resource Attributes
+
+The AWS Container Insights Processor decorates metrics differently depending on their types.
+
+### Cluster metrics
+No decoration.
+
+### Cluster Namespace metrics
+No decoration.
+
+### Cluster Service metrics
+No decoration.
+
+
+### Node metrics
+| Added Metric                      | Unit      |
+|-----------------------------------|-----------|
+| node_cpu_request                  | Millicore |
+| node_cpu_reserved_capacity        | Percent   |
+| node_memory_request               | Bytes     |
+| node_memory_reserved_capacity     | Percent   |
+| node_number_of_running_containers | Count     |
+| node_number_of_running_pods       | Count     |
+
+The following resource attributes are added:
+* Sources
+* kubernetes 
+
+
+### Node Disk IO metrics
+The following resource attributes are added:
+* Sources
+* kubernetes
+
+
+
+### Node Filesystem metrics
+The following resource attributes are added:
+* Sources
+* kubernetes
+
+
+### Node Network metrics
+The following resource attributes are added:
+* Sources
+* kubernetes
+
+
+### Pod metrics
+| Added Metric                          | Unit      | 
+|---------------------------------------|-----------|
+| pod_cpu_limit                         | Millicore |
+| pod_cpu_request                       | Millicore |
+| pod_cpu_reserved_capacity             | Percent   |
+| pod_cpu_utilization_over_pod_limit    | Percent   |
+| pod_memory_limit                      | Bytes     |
+| pod_memory_request                    | Bytes     |
+| pod_memory_reserved_capacity          | Percent   |
+| pod_memory_utilization_over_pod_limit | Percent   |
+| pod_number_of_container_restarts      | Count     | 
+| pod_number_of_containers              | Count     |   
+| pod_number_of_running_containers      | Count     |  
+
+The following resource attributes are added:
+* Sources
+* kubernetes
+* pod_status
+
+
+
+### Pod Network metrics
+The following resource attributes are added:
+* Sources
+* kubernetes
+* pod_status
```

Comment:
Where is the list of Pod network metrics? Are they missing or we only add resource attributes here?

Reply:
We are just adding resource attributes.